group_vars/figgy_staging.yml

```diff
@@ -33,6 +33,7 @@ figgy_rabbit_user: '{{vault_figgy_staging_rabbit_user}}'
 figgy_rabbit_password: '{{vault_figgy_staging_rabbit_password}}'
 figgy_rabbit_host: 'figgy-staging1.princeton.edu'
 figgy_rabbit_server: 'amqp://{{figgy_rabbit_user}}:{{figgy_rabbit_password}}@{{figgy_rabbit_host}}:5672'
+figgy_repository_mount: '/mnt/diglibdata/hydra_binaries/figgy_staging'
```

Comment:
Does this have an issue? Doesn't seem related to ruby / bundler versions.

Reply:
It doesn't, I just couldn't run the playbook without it.

Reply:
Seems important. I wonder how it came to be missing. Could you split it into its own commit, at least?

Reply:
Sure.